backend/src/controllers/booking/deleteBookingController.ts

```diff
@@ -14,8 +16,7 @@ export const deleteBooking = () => {
         next: NextFunction
     ) => {
         try {
-            const bookingId: string = req.path.substring(1);
-
+            const bookingId: string = req.params.bookingId;
```

Comment:
Sain sellaisen kuvan Expressin dokumentaatiosta, että URL:ssä olevat parametrit kuuluisi lukea näin, niin muutin tähänkin

Reply:
Joo hyvä muutos ja toimii sitten tulevaisuudessakin kun on dokumentaation mukainen